helm/ray/values.yaml.tpl

```diff
@@ -43,7 +43,7 @@ head:
     #   requests:
     #     cpu: "500m"
     #     memory: "512Mi"
-  init_container_enabled: true
+  init_container_enabled: false
```

Comment:
this might affect syntho-cli I guess 🤔 I'll need to double check if cli is able deploy from this branch's version

Reply:
Just out of curitosity, shy it should be `false` as a default value?

Reply:
It could default to `true` as well, but as far as I know, the running of this `chmod` command is not necessary unless permission issues happen. This comment also explains why this part might also not be necessary if fixed in another way: https://github.com/syntho-ai/deployment-tools/pull/6#discussion_r1604779508.

Reply:
AFAIR, as a default behavior, I think the cli can't deploy without this to be true. I remember I hit this issue in the past and I made it `true` in the .tpl file, cuz the oriignal values still has it as `false`. Let me make sure if this is the case, and will get back to you.

Reply:
Did you check this btw already for the Syntho CLI? Not an issue if it needs to be true by default, it's just making sure certain permissions are there

Reply:
I am testing it now

Reply:
tests have been passed successfully, so it looks like it is safe to do it.